### ISSUE TITLE
See Repo for Code Revision History

### ISSUE THREAD

**Issue by zakski** [sirus20x6/stargen#13]
https://github.com/zakski/accrete-starform-stargen

**Comment by sirus20x6** [comment]
yes, you should add this one to the list. I've modernized enough of the code that it happily compiles with new tools
